tiler/app/middlewares/url_injector.py

```diff
@@ -31,7 +31,7 @@ def s3_presigned_access(url: str | None = Query(default=None, description="Optio
     """
     if not url:
         if not default_cog:
-            raise Exception("DEFAULT_COG env var is not set. It is required if no URL is paased to the tiler")
+            raise Exception("DEFAULT_COG env var is not set. It is required if no URL is passed to the tiler")
```

Comment:
a little bit nit pick but this could be `ValueError` instead of a plain `Exception`